src/pkg/artifact/dao/model.go

```diff
@@ -30,7 +30,7 @@ func init() {
 // Artifact model in database
 type Artifact struct {
 	ID                int64     `orm:"pk;auto;column(id)"`
-	Type              string    `orm:"column(type)"`                // image or chart
```

Comment:
Pls keep the previous comments, the artifact is used for oci registry, not the chartmuseum artifact,  the type could be chart if it is a chart with OCI artifact.